tjpcov/covariance_cluster_counts_gaussian.py

```diff
@@ -84,15 +83,12 @@ def shot_noise(self, z_i, lbd_i):
         Returns:
             float: Gaussian covariance contribution
         """
+
         # Eqn B.7 or 1601.05779.pdf eqn 1
         def integrand(z):
-            return (
```

Comment:
I realized we can use the existing base class methods instead of re-implementing them.  Unit tests are still passing.  Before we were duplicating code basically.